data/mods/innawood/recipes/tools_hand.json

```diff
@@ -127,6 +127,30 @@
       [ [ "stick", 4 ], [ "2x4", 2 ] ]
     ]
   },
+  {
+    "result": "bronze_sickle",
+    "type": "recipe",
+    "activity_level": "BRISK_EXERCISE",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_TOOLS",
+    "skill_used": "fabrication",
+    "difficulty": 5,
+    "time": "5 h",
+    "autolearn": true,
+    "proficiencies": [
+      { "proficiency": "prof_metalworking" },
+      { "proficiency": "prof_redsmithing" },
+      { "proficiency": "prof_toolsmithing" }
+    ],
+    "using": [ [ "forging_standard", 2 ], [ "bronzesmithing_tools", 1 ] ],
+    "qualities": [ { "id": "GRIND", "level": 2 } ],
+    "tools": [ [ [ "casting_mold", -1 ] ] ],
+    "components": [
+      [ [ "leather", 1 ], [ "fur", 1 ], [ "cordage_short", 2, "LIST" ], [ "filament", 100, "LIST" ] ],
+      [ [ "scrap_bronze", 5 ] ],
+      [ [ "stick", 2 ], [ "2x4", 1 ] ]
```

Comment:
Seems to me one stick would be enough wood to make a typical sickle handle. But if not, perhaps allow the use of a long stick as an alternative to two short ones?

Reply:
Oh, yeah, totally. As its supposed do be handheld, its just a short handle. I did it quick and dirty this morning and did not put much thought into it. ^^'